config.dev.toml

```diff
@@ -99,6 +99,10 @@ EnablePprof = false
 # PprofPort specifies the port on which pprof will be exposed if activated.
 PprofPort = ":3001"
 
+# ConfigPort specifies the port on which the /config endpoint will listen for
+# providing the internal configuration
```

Comment:
Let's add a note about secrets: "Note if you have secrets in your configuration, this endpoint will return them in plaintext. This is why they are exposed in a different port and should be used for debugging purposes only".